arch/arm/core/aarch64/mmu/arm_mmu.c

```diff
@@ -272,15 +287,6 @@ static const struct arm_mmu_region mmu_zephyr_regions[] = {
 			      (uintptr_t)_image_rodata_start,
 			      (uintptr_t)_image_rodata_size,
 			      MT_NORMAL | MT_P_RO_U_NA | MT_SECURE),
-
-	/* Mark rest of the zephyr execution regions (data, bss, noinit, etc.)
```

Comment:
unnecessary diff generated.

Reply:
I purposely moved `zephyr_data` close to the SRAM region as a reminder that the two regions are mostly overlapping.

Reply:
ok